[2019-11-16T14:10:55.922] [ERROR] application - TypeError: Date(...).getSeconds is not a function
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:63:25)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[2019-11-16T14:13:19.566] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T14:25:29.559] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T14:31:57.994] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T14:37:53.854] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T14:41:17.644] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T14:41:29.561] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T17:02:13.519] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T17:09:11.250] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T17:09:17.833] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
[2019-11-16T17:09:23.401] [ERROR] application - { Error: 打卡失败
    at Function.attence_attend (/Users/ios/JJGit/NodeJs/Koa/token/modules/app/app.js:68:13)
    at router.post (/Users/ios/JJGit/NodeJs/Koa/token/routers/apis/app.js:28:13)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-router/lib/router.js:351:31)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at /Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-log4/koa-logger.js:86:13
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at module.exports (/Users/ios/JJGit/NodeJs/Koa/token/core/exception.js:5:11)
    at dispatch (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/ios/JJGit/NodeJs/Koa/token/node_modules/koa-bodyparser/index.js:86:11)
    at process._tickCallback (internal/process/next_tick.js:68:7) status: -1, msg: '打卡失败' }
